k8s-artifacts/mg-enforcer/deployment.yaml

```diff
@@ -0,0 +1,76 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Deployment for enforcer
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mg-enforcer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mg-enforcer
+  template:
+    metadata:
+      labels:
+        app: mg-enforcer
+    spec:
+      containers:
+        - name: mg-enforcer
+          volumeMounts:
+            - mountPath: /home/wso2/mg/security/
+              name: client-truststore-vol
+            - mountPath: /home/wso2/conf/config.toml
+              subPath: config.toml
+              name: config-toml-vol
+            - mountPath: /home/wso2/conf/log4j2.properties
+              subPath: log4j2.properties
+              name: log4j2-vol
+            - mountPath: /home/wso2/conf/log_config.toml
```

Comment:
Lets remove this mount from enforcer since it is used by adapter only

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf